WordPress/src/main/java/org/wordpress/android/ui/prefs/SiteSettingsFragment.java

```diff
@@ -1123,6 +1131,26 @@ private void updateWhitelistSettings(int val) {
                 getWhitelistSummary(val));
     }
 
+    private void handleStartOver() {
+        // Only paid plans should be handled here, free plans should be redirected to website from "Start Over" button
+        if (mSite == null || mSite.getHasFreePlan()) {
+            return;
+        }
+        Intent intent = new Intent(Intent.ACTION_SEND);
+        intent.setType("text/plain");
```

Comment:
You can eliminate a lot of the noise in the system Activity picker by using `ContactsContract.CommonDataKinds.Email.CONTENT_ITEM_TYPE` here instead of `text/plain`.

See [this StackOverflow question](http://stackoverflow.com/questions/5855022/android-pick-email-intent) for more details.